CMakeLists.txt

```diff
@@ -262,6 +262,9 @@ cxx_feature_check(STEADY_CLOCK)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+# Include 3rd party module gflags
+include(GFlags)
```

Comment:
Thank you for bothering writing *this* logic, i would have insisted on it if it wasn't there,
but by default it should not be doing that, but relying on pre-installed version,
that should be looked for via traditional `find_package(GFlags)`.